### PR TITLE
Several changes to rocket and leaderboard

### PR DESCRIPTION
**Pull request by charasyn**
JIRA issues:
- FGTS-205: Scoreboard doesn't show if player doesn't get high score
  Fixed, player score will show at bottom and play sad sfx

- FGTS-209: Scoreboard doesn't display properly on final level
  Weird issue, I was able to reproduce on other levels.
  Fixed by ensuring we don't accidentally add too many scores on screen

Other changes:
- Prevent scoreboard from changing while the victory sfx is playing
  Only affects Level3.tscn.
  The rocket gets the high score screen and sets `wait_in_initial_state`

- Rocket boom sfx file changed to incorporate the distortion
  Something didn't sit right with me with amplifying it by 24dB in-game.
  I messed around with the sfx file in my audio editor and got it to
  sound "boomy" enough without actually being loud.

- Rocket boom sound nodes adjusted
  Before, we had one 2D sound node in the middle of the rocket.
  This resulted in the sound being positionally weird.
  (Ex. the left rocket ignited but the sound came from the right)
  Now, we have 3 separate sound nodes: left, right, and non-positional.
  The non-positional one is used for the rocket taking off at the end.

### CODE REVIEW

File: godotproject/Objects/Rocket/Rocket.gd [hunk 1]
```diff
@@ -59,17 +59,21 @@ func go_to_space():
...
+	$JetBoomNonLocational.play()
 	$Timer.start()
 	
 	
 func _on_Timer_timeout():
 	# Rocket has now left
 	if frog_on_board:
 		$VictorySoundPlayer.play()
+		# Prevent high score screen from going until the sound effect is done
+		Global.get_node("HighScoreScreenLayer/HighScoreScreen").wait_in_initial_state = true
```
Comment (fheysel) [inline godotproject/Objects/Rocket/Rocket.gd]: Can we add this to every highscore screen and not just at the end of the game? I know I was the one to implement this in the first place, I have it listed as a TODO under my polish all levels ticket, I just dont want to get merge conflicts if we both touch the highscore script

Comment (charasyn) [reply godotproject/Objects/Rocket/Rocket.gd]: This being the Victory sound? We could def do that

Comment (fheysel) [reply godotproject/Objects/Rocket/Rocket.gd]: yeah

**Review by fheysel — APPROVED** [review]
Looks good to me.

**Pull request merged**